packages/qwik/src/server/prefetch-strategy.ts

```diff
@@ -78,7 +79,7 @@ function addBundle(
   buildBase: string,
   bundleFileName: string
 ) {
-  const url = buildBase + bundleFileName;
+  const url = qDev ? bundleFileName : buildBase + bundleFileName;
```

Comment:
I think this also needs the qrl parent, see #6936 and https://github.com/QwikDev/qwik/blob/6ba3052817979c1b24a25f3a113bbcd0849fa71c/packages/qwik/src/optimizer/src/plugins/vite-dev-server.ts#L76

Reply:
The browser is able to load the file without the _qrl_parent query params. Why do we need this query params when loading the module ?